src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__card card" data-qa="card">
+
+        <img 
+          class="card__img card__img--margin-bottom"
+          src="./images/imac.jpeg" 
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <div class="info-width">
+          <div class="product-info product-info--margins-bottom">
+            <p class="product-info__model">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+            <p class="product-info__code">Product code: 195434</p>
+          </div>
+  
+          <div class="rating rating--margin-bottom">
```

Comment:
![image](https://user-images.githubusercontent.com/97547824/211195370-180013a0-9501-4d26-a3d2-35db8133f2c0.png)

Reply:
Fix it everywhere